.github/workflows/manifold.yml

```diff
@@ -100,6 +100,15 @@ jobs:
       run: |
         cd build/test
         node ./manifold_test.js
+    - name: 'Upload WASM files'
+      uses: actions/upload-artifact@v3
```

Comment:
I guess we just need to add `actions/download-artifact` to `deploy.yml` and then add that output path to this list: https://github.com/elalish/manifold/blob/master/deploy.sh#L23, which copies is to `docs/html`, which in turn becomes the root of our served github pages. I guess the shell script will need to take a couple of input paths.

Reply:
Yes, I think maybe you can modify the `deploy.yml` as I'm not familiar with this.

Reply:
Sure; I can take over this PR if you like. Thanks for pointing me to that action.